.git2gus/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "productTag": "a1aB00000004Bx8IAE",
+    "defaultBuild": "offcore.tooling.50.10.0",
```

Comment:
@amphro This seems like it will be a maintenance issue that needs to be remembered.